src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -44,19 +53,44 @@ public Converter721To730(ICompoundFactory compoundFactory, ICloner cloner, ICrea
 
       public void Visit(Compound compound)
       {
+         resetAllParametersToIsDefaultIn(compound);
          convertCompound(compound);
       }
 
       public void Visit(Simulation simulation)
       {
+         simulation.AllBuildingBlocks<IPKSimBuildingBlock>().Each(resetAllParametersToIsDefaultIn);
```

Comment:
Converting all building blocks of the simulation as well as the simulation itself.

That being said, compound parameter IN the simulation (such as Lipo) will not show up as changed.
However newly created simulation with old compound will show the changes. I think this is good enough as there is NO way we can catch all parameters that have changed in the simulation anyways. Better show only the one that really changed in the simulation so that it's clearer